diagnostic.md

```diff
@@ -73,14 +88,28 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request POST
```

Comment:
Should look something like this: 
`curl --include --request "POST" "http://example.com/donuts" \
  --header "Content-Type: application/json" \
  --data $'{
    "donut": {
      "name": "French Cruller",
      "price": "$0.99"
    }
  }'`